runtime/compiler/trj9/optimizer/EscapeAnalysis.cpp

```diff
@@ -5771,7 +5771,7 @@ void TR_EscapeAnalysis::avoidStringCopyAllocation(Candidate *candidate)
  */
 bool TR_EscapeAnalysis::tryToZeroInitializeUsingArrayset(Candidate* candidate, TR::TreeTop* precedingTreeTop)
    {
-   if (cg()->getSupportsArraySetToZero())
+   if (cg()->getSupportsArraySet())
```

Comment:
Was getSupportsArraySetToZero always returning true when getSupportsArraySet was set to true - I don't see an obvious aliasing in the flags. If it wasn't then there is a perf question around this change since it will cause this code to kick in more frequently...

Reply:
Yes, currently getSupportsArraySetToZero() always returns same as getSupportsArraySet(); in other words, the two flags are either both true or both false.